### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by mrhota**
Hi again! just bumping some deps and cleaning some warnings before I jump back in. Hope you're doing well!

### CODE REVIEW

**Comment by mrhota** [comment]
~@anp apparently lots of the recent compiler refactors changed the way object files get named in the `.a` output file! so our `frankenstein.sh` script might not work the way it used to.~

never mind, I think I spoke prematurely.

**Comment by anp** [comment]
Hi! Looks like your malloc changes here are just syntactic, maybe upstream musl fixed some previously failing tests?

**Comment by mrhota** [comment]
oh yeah, I just noticed changes to `mmap` which I think broke these malloc tests.

**Comment by mrhota** [comment]
@anp ok, I'll call this PR done now. What do you think?

File: scripts/examine.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#    0:   48 89 f2                mov    %rsi,%rdx
+#    3:   31 f6                   xor    %esi,%esi
+#    5:   e9 00 00 00 00          jmpq   a <bzero+0xa>
+#    0:
+
+fn=$1; shift 1
+exec objdump -d "$@" |
+awk " /^[[:xdigit:]].*<$fn>/,/^\$/ { print \$0 }" |
+awk -F: -F' '  'NR==1 {  offset=strtonum("0x"$1); print $0; }
+                NR!=1 {  split($0,a,":"); rhs=a[2]; n=strtonum("0x"$1); $1=sprintf("%x", n-offset); printf "%4s:%s\n", $1,rhs }'
```
Comment (mrhota) [inline scripts/examine.sh:17]: copied from stackoverflow!

**Review by anp — APPROVED** [review]
Awesome!

**Comment by mrhota** [comment]
@anp I don't have merge permission, so you'll have to merge/squash these PRs

**Comment by anp** [comment]
Oops!

**Pull request merged**